src/pages/EnvironmentsPage.test.ts

```diff
@@ -84,9 +84,12 @@ describe('Environments Page navProvider', () => {
     pageNav.getUrl('workspace-view', { name: 'myName', namespace: 'myNamespace' });
 
     // Assert
-    const watcher = asMockedFn(mockNav.getLink);
-    expect(watcher).toBeCalledTimes(1);
-    expect(watcher).toBeCalledWith(terraNavKey('workspace-dashboard'), { name: 'myName', namespace: 'myNamespace' });
+    const getLinkWatcher = asMockedFn(mockNav.getLink);
```

Comment:
Nit: It'd be nice to get rid of the inconsistency where some mocks are named with a "watcher" suffix.

This could be addressed by modifying the "arrange" section to:
```ts
const getLink = jest.fn();
const goToPath = jest.fn();
const mockNav: TerraNavLinkProvider = { getLink, goToPath };
```

Reply:
asMockedFn adds a type check to make sure mockNav.getLink is in fact a jest mock function... but running the test will produce an actionable error if this is violated... 

Let me try this out...